apps/api/src/modules/notifications/entities/notification.entity.ts

```diff
@@ -81,6 +81,10 @@ export class Notification {
   @Field()
   applicationId: number;
 
+  @Column({ name: 'retry_count', type: 'int', default: 0 })
+  @Field()
+  retryCount: number;
```

Comment:
Once confirm that create notification dto doesnt allow updating this value through API

Reply:
CreateNotificationDto class only has the below properties:

```ts
export class CreateNotificationDto {
  @IsNumber()
  providerId: number;

  @IsObject()
  @IsDataValid()
  data: Record<string, unknown>;
}
```

So we can't really pass the retryCount through that API.

We also have te GraphQL query for getting all notifications, but that is a Query, not Mutation, so that cannot be used for updating retry count either.